dissect/target/plugins/filesystem/walkfs.py

```diff
@@ -28,6 +28,8 @@
         ("uint32", "uid"),
         ("uint32", "gid"),
         ("string[]", "fstypes"),
+        ("string[]", "vuuid"),
+        ("string[]", "dserial"),
```

Comment:
```suggestion
        ("string[]", "disk_identifiers"),
```
maybe a bit more clear what the field is for